src/schemas.jl

```diff
@@ -251,48 +251,82 @@ otherwise constitutes type piracy and should be avoided.
 accepted_field_type(::SchemaVersion, ::Type{UUID}) = Union{UUID,UInt128}
 accepted_field_type(::SchemaVersion, ::Type{Symbol}) = Union{Symbol,String}
 
+
+
 """
     Legolas.find_violation(ts::Tables.Schema, sv::Legolas.SchemaVersion)
 
-For required field `f::F` of `sv`:
+Return first schema violation; otherwise, return `nothing`. For description of
+schema violations, see [`Legolas.find_violations`](@ref).
 
-- Define `A = Legolas.accepted_field_type(sv, F)`
-- If `f::T` is present in `ts`, ensure that `T <: A` or else immediately return `f::Symbol => T::DataType`.
-- If `f` isn't present in `ts`, ensure that `Missing <: A` or else immediately return `f::Symbol => missing::Missing`.
+See also: [`Legolas.find_violations`](@ref), [`Legolas.validate`](@ref), [`Legolas.complies_with`](@ref)
```

Comment:
this docstring specification of what constitutes a "violation" should be kept as-is - it's an important part of how Legolas operates

it's also IMO a bit easier to parse and more precise than the more prose-y version added in `find_violations` (and makes the connection to `accepted_field_type` clear/obvious)

Reply:
Done